src/app/submission/form/submission-form.component.ts

```diff
@@ -128,7 +135,11 @@ export class SubmissionFormComponent implements OnChanges, OnDestroy {
           } else {
             return observableOf([]);
           }
-        }));
+        }),
+        tap((sectionList) => {
+          this.uploadEnabled$.next(isNotEmpty(sectionList) && sectionList.some(config => config.sectionType === SectionsType.Upload));
```

Comment:
In my opinion a cleanest way to find out if the upload section is available is to use the [sections.service](https://github.com/DSpace/dspace-angular/blob/main/src/app/submission/sections/sections.service.ts). it includes a [method to check the availability of a section by given id](https://github.com/DSpace/dspace-angular/blob/ea3886bbfa63d8bc41664f3d76d0534863faceac/src/app/submission/sections/sections.service.ts#L303-L310). It'd would be nice to have a similar method that use the section type to check if any sections with the given type are available